src/java.desktop/share/classes/java/awt/image/ColorConvertOp.java

```diff
@@ -690,7 +690,7 @@ private int getRenderingIntent (ICC_Profile profile) {
          * shall be set to zero. Thus, we are ignoring two most significant
          * bytes here.
          *
-         *  See http://www.color.org/ICC1v42_2006-05.pdf, section 7.2.15.
+         *  See https://www.color.org/ICC1v42_2006-05.pdf, section 7.2.15.
```

Comment:
@prrace Current ICC doc links point to older ICC Specification document. Do they need to be updated to the latest one?
https://www.color.org/specification/ICC.1-2022-05.pdf

Reply:
That would be out of scope for this.